src/cowSdk.ts

```diff
@@ -3,5 +3,10 @@ import { OrderBookApi } from '@cowprotocol/cow-sdk'
 
 import { isBarn, isDev, isLocal, isPr } from 'legacy/utils/environments'
 
+// TODO: find a solution for an issue with environment problem with TWAP
+const isTwapTesting = true
```

Comment:
https://github.com/cowprotocol/cowswap/pull/2597/commits/3062d99cc311f24c0b8b92ca22ef48dd53f30547
Here I've tried to isolate environment conflict in TWAP module scope. Unfortunately, it doesn't work well :( 
So, I just hard-coded prod env for the whole App.
I've no idea how to properly handle this

Reply:
Is it a problem if the app remains with the current behaviour and only when fetching twap orders it reads from prod?

Otherwise, can we add a flag to the local Order obj indicating whether it needs to force a given backend env?

Reply:
Fixed it:
https://github.com/cowprotocol/cowswap/pull/2597/commits/f4338bebe7306fc6925b512819cc0e411566b75c

Now we load discrete orders from PROD only when current env is not PROD

Reply:
Great! 

But I left a comment regarding the implementation, just to make sure we don't overcomplicate it https://github.com/cowprotocol/cowswap/commit/f4338bebe7306fc6925b512819cc0e411566b75c#r117050932